packages/wallets/src/features/cashier/modules/Transactions/Transactions.tsx

```diff
@@ -78,6 +79,7 @@ const Transactions = () => {
                     icon={<FilterIcon />}
                     label='Filter'
                     list={filterOptionsList}
+                    maxWidth={isMobile ? 'unset' : '19.5rem'}
```

Comment:
as per other comments and discussion from slack - remove inline from .jsx and styles to **style**sheets? 

```suggestion
                    maxWidth={isMobile ? 'unset' : '19.5rem'}
```

Reply:
CSS override is being overriden by the component prop:

<img width="1321" alt="Screenshot 2023-11-15 at 17 51 59" src="https://github.com/binary-com/deriv-app/assets/119863957/acbcfcce-e27d-4cfd-ad11-bfbe159d667b">

Suppose we should do such overrides solely in CSS right?

Reply:
yes, we should not apply inline styling in .jsx, thats like going back in time when .css didn't even existed,

thats exactly why I'm commenting so much about that ever since I noticed - the more inline styles there is gonna be, the harder its gonna go back and avoid that approach, as its clearly visible even now with relatively small change :(

Reply:
so now the only option I see to override inline styles which are already added there is the use of !important... which is also pretty unpleasant... but at least enables us to stop the pattern of adding inline styles from spreading all over codebase

Reply:
actually, could we just make sure that WalletDropdown component does not apply any default inline maxWidth when prop is not present? existing defulat value could be moved to .css

Reply:
Done